tests/foreman/cli/test_activationkey.py

```diff
@@ -1698,7 +1698,7 @@ def test_negative_update_auto_attach(self):
         except CLIFactoryError as err:
             self.fail(err)
 
-        self.assertIn("'--auto-attach': value must be one of", result.stderr)
+        self.assertIn(u"'--auto-attach': value must be one of", result.stderr)
```

Comment:
This seems like a great use case for double quotes, to me. Unlike a language like sh, there's no syntactic difference between single and double quotes. So when I see double quotes, I think "there must be some specific reason double quotes are used." And so it is.